checkpointctl.go

```diff
@@ -93,19 +94,25 @@ func setupInspect() *cobra.Command {
 		&mounts,
 		"mounts",
 		false,
-		"Display an overview of mounts used in the checkpoint",
+		"Display an overview of mounts used in the container checkpoint",
 	)
 	flags.BoolVar(
 		&psTree,
 		"ps-tree",
 		false,
 		"Display an overview of processes in the container checkpoint",
 	)
+	flags.BoolVar(
+		&files,
+		"files",
+		false,
+		"Display an overview of files used in the container checkpoint",
+	)
 	flags.BoolVar(
 		&showAll,
 		"all",
 		false,
-		"Show all information about container checkpoint",
+		"Show all information about the container checkpoint",
```

Comment:
We intentionally don't use "the" here because it would suggest that this option is designed to show information about a particular checkpoint. By omitting "the", we convey that `--all` is capable of providing information about any container checkpoint, which is aligned with the functionality supporting multiple checkpoints, e.g., `checkpointctl inspect --all *.tar.gz`.

Reply:
Got it. In that case, "checkpoints" would make more sense.